pkg/oonf_api/0002-port-tests-to-riot.patch

```diff
@@ -189,18 +189,17 @@ index 0000000..8bf9faa
 +	cp $test\.c $test
 +done
 diff --git a/tests/rfc5444/test_rfc5444_reader_dropcontext.c b/tests/rfc5444/test_rfc5444_reader_dropcontext.c
-index df203fd..29daff6 100644
+index df203fd..0a305e3 100644
 --- a/tests/rfc5444/test_rfc5444_reader_dropcontext.c
 +++ b/tests/rfc5444/test_rfc5444_reader_dropcontext.c
-@@ -38,7 +38,7 @@
-  * the copyright holders.
+@@ -39,6 +39,7 @@
   *
   */
--#include <arpa/inet.h>
+ #include <arpa/inet.h>
```

Comment:
No. I don't even know why it was there in the first place... Older versions of RIOT had this header too.